src/csharp/Microsoft.Spark/Sql/GenericRow.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql.Types;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Represents a row object in RDD, equivalent to GenericRow in Spark.
+    /// </summary>
+    public sealed class GenericRow
```

Comment:
removed sealed.   Have Row type subclass this.  Remove common methods from Row.cs

Reply:
On making Row as subclass of GenericRow, it gives error as Row takes in schema in constructor

Reply:
can we not remove `sealed` unless you intend to be subclassed outside this project? You can use composition if you don't want to duplicate the implementation.

Reply:
That should be okay too.  You can create a genericRow field within Row and call the genericRow methods where appropriate.

Reply:
I'll remove the sealed

Reply:
Done. thanks!